exploratory_analysis.ipynb

```diff
@@ -1709,28 +1900,28 @@
       ],
       "text/plain": [
        "       state           county  state_code  county_code        measure_name  \\\n",
-       "257226    TN    Shelby County        47.0        157.0  Violent crime rate   \n",
-       "254638    OK     Adair County        40.0          1.0  Violent crime rate   \n",
-       "254580    OH  Van Wert County        39.0        161.0  Violent crime rate   \n",
-       "254587    OH    Vinton County        39.0        163.0  Violent crime rate   \n",
-       "254593    OH    Warren County        39.0        165.0  Violent crime rate   \n",
+       "245430    IN     Boone County        18.0         11.0  Violent crime rate   \n",
+       "247672    KY     Lewis County        21.0        135.0  Violent crime rate   \n",
+       "247732    KY  Magoffin County        21.0        153.0  Violent crime rate   \n",
+       "247725    KY   Madison County        21.0        151.0  Violent crime rate   \n",
+       "247718    KY    McLean County        21.0        149.0  Violent crime rate   \n",
        "\n",
-       "        measure_id     numerator   denominator    raw_value  fips_code  \\\n",
-       "257226        43.0  11742.333330  924734.66670  1269.805681    47157.0   \n",
-       "254638        43.0     76.000000   22546.00000   337.088619    40001.0   \n",
-       "254580        43.0     36.666667   28748.66667   127.542147    39161.0   \n",
-       "254587        43.0     16.666667   13417.66667   124.214344    39163.0   \n",
-       "254593        43.0    154.666667  208683.33330    74.115486    39165.0   \n",
+       "        measure_id   numerator   denominator   raw_value  fips_code  \\\n",
+       "245430        43.0    9.500000  40952.000000   23.197890    18011.0   \n",
+       "247672        43.0    9.000000  13877.000000   64.855516    21135.0   \n",
+       "247732        43.0    4.666667  13304.000000   35.077170    21153.0   \n",
+       "247725        43.0  168.666667  83345.666670  202.370049    21151.0   \n",
+       "247718        43.0    2.666667   9597.333333   27.785496    21149.0   \n",
        "\n",
        "        start_year  \n",
-       "257226      2009.0  \n",
-       "254638      2009.0  \n",
-       "254580      2009.0  \n",
-       "254587      2009.0  \n",
-       "254593      2009.0  "
+       "245430      2009.0  \n",
+       "247672      2009.0  \n",
+       "247732      2009.0  \n",
+       "247725      2009.0  \n",
+       "247718      2009.0  "
       ]
      },
-     "execution_count": 68,
+     "execution_count": 163,
```

Comment:
These aren't the top 5 measures with highest raw_value because you're sorting by 'measure_name' in descending order, so it is the top 5 rows with measure_name closest to z in alphabet. I think it may be better to group by measure_name taking the max raw_value for each group, then sort by the max raw_value in descending order? I'm also interested in the non-null raw_value count for each measure within each year group